utils/s3/client.cc

```diff
@@ -82,6 +81,19 @@ future<> ignore_reply(const http::reply& rep, input_stream<char>&& in_) {
     co_await util::skip_entire_stream(in);
 }
 
+future<> look_for_errors(const http::reply&, input_stream<char>&& in_) {
+    auto in = std::move(in_);
+    auto body = co_await util::read_entire_stream_contiguous(in);
```

Comment:
Doc say:

> While processing is in progress, Amazon S3 periodically sends white space characters to keep the connection from timing out ...

Are these space chars handled correctly by aws_error::parse()?

Reply:
https://github.com/scylladb/scylladb/pull/20518/files#diff-6175bdb198789e5b4838dcc16479916dcb7c97839ca3e2e2aef0c5f607fef7c6R65
and
https://github.com/scylladb/scylladb/pull/20518/files#diff-6175bdb198789e5b4838dcc16479916dcb7c97839ca3e2e2aef0c5f607fef7c6R71

Reply:
> https://github.com/scylladb/scylladb/pull/20518/files#diff-6175bdb198789e5b4838dcc16479916dcb7c97839ca3e2e2aef0c5f607fef7c6R65 

This parses `""` -- and empty string. No spaces

> and https://github.com/scylladb/scylladb/pull/20518/files#diff-6175bdb198789e5b4838dcc16479916dcb7c97839ca3e2e2aef0c5f607fef7c6R71

This parses some random text.

Both parse into OK, while I was asking more about a request that contains several spaces, then an XML with real error

Reply:
oh, I see, lemme add another test